### ISSUE TITLE
Patch for /trunk/pyguide.html

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/google-styleguide#6]
```
Wrong function name in indirect setter
```

---

Original issue reported on code.google.com by `overk...@gmail.com` on 23 Jun 2011 at 8:56

Attachments:
- [pyguide.html.patch](https://storage.googleapis.com/google-code-attachments/google-styleguide/issue-6/comment-0/pyguide.html.patch)

**Comment by GoogleCodeExporter** [comment]
```
This is fixed in the Python style guide 2.29 release at r76.
```

---

Original comment by `mark@chromium.org` on 4 Nov 2011 at 9:14
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
---

Original comment by `mark@chromium.org` on 4 Nov 2011 at 9:18
